webextension/js/popup.js

```diff
@@ -552,7 +568,7 @@ Logic.registerPanel(P_CONTAINER_EDIT, {
     return browser.runtime.sendMessage({
       method: "createOrUpdateContainer",
       message: {
-        userContextId: identity.userContextId || false,
+        userContextId: Logic.userContextId(identity.cookieStoreId) || false,
```

Comment:
Some of these can probably be cleared up later when we have all the code ported over. I just want to reduce change as much as possible. Neatening etc can happen later.